backend/de.metas.externalsystem/src/main/java/de/metas/externalsystem/service/IExternalSystemConfigDAO.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * #%L
+ * de.metas.externalsystem
+ * %%
+ * Copyright (C) 2021 metas GmbH
+ * %%
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as
+ * published by the Free Software Foundation, either version 2 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public
+ * License along with this program. If not, see
+ * <http://www.gnu.org/licenses/gpl-2.0.html>.
+ * #L%
+ */
+
+package de.metas.externalsystem.service;
+
+import de.metas.externalsystem.ExternalSystemParentConfig;
+import de.metas.rest_api.utils.MetasfreshId;
+import de.metas.util.ISingletonService;
+
+public interface IExternalSystemConfigDAO extends ISingletonService
+{
+	ExternalSystemParentConfig getById(MetasfreshId id);
```

Comment:
please change to AlbertaConfigId or something. bc it's otherwise not clear if this is about the alberta-config's or the parent-record's ID

Reply:
Added ExternalSystemConfigId.